src/neuroconv/datainterfaces/ecephys/openephys/openephyslegacydatainterface.py

```diff
@@ -43,15 +45,22 @@ def __init__(
         ----------
         folder_path : FolderPathType
             Path to OpenEphys directory.
-        stream_id : str, default: "CH"
+        stream_id : str, default: None
             The identifier of the recording stream (e.g. "CH" for channel data).
+            When the recording stream is not specified it uses the first recording stream.
         stream_name : str, default: None
         block_index : int, default: 0
         all_annotations : bool, default: False
         stub_test : bool, default: False
         verbose : bool, default: True
         """
         self.RX = OpenEphysLegacyRecordingExtractor
+
+        self.folder_path = folder_path
+        if stream_name is None and stream_id is None:
+            stream_channels = self._get_stream_channels()
+            stream_id = list(stream_channels["id"])[0]
```

Comment:
Notes from @bendichter >
if  "CH" is present in signal streams then choose it, if not possible can return list of possible stream ids